explainer.md

```diff
@@ -341,70 +348,77 @@ input.accessibleNode.activeDescendant = optionList.accessibleNode;
 
 This would allow the relationship to be expressed naturally.
 
-#### No reflection
+#### AOM and ARIA
```

Comment:
It may be obvious but should we explicitly define AOM the first time before using it as an acronym?